### DISCUSSION TITLE
鹅鸭

### DISCUSSION THREAD

**Discussion by 1066258184** [FrozenFish259/Goose_Goose_Duck_Hack#285] [General 聊天区]
如果能修复看身份牌就好了，我家庭版没有沙盒，要专业版，谢谢

**Comment by FrozenFish259** [comment]
sandboxie-plus

**Reply by 1066258184** [reply]
好的晚上的时候试了一下sandboxie-plus x84不行明天试试x64，ARM。还有群里人满了，可不可以在创一个方便交流打赏，谢谢，一直支持，早点休息，身体最重要

**Reply by FrozenFish259** [reply]
嗯谢谢支持，首页有赞助按钮，群的话下载发行版，压缩包里也有图片

**Reply by 1066258184** [reply]
图片全满了

**Reply by 1066258184** [reply]
大佬，本机玩游戏还是沙盒，我用本机玩，辅助没效果显示的是沙盒的
